backend/src/test/java/io/debezium/configserver/ValidatePostgresFiltersIT.java

```diff
@@ -0,0 +1,103 @@
+package io.debezium.configserver;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.debezium.configserver.rest.ConnectorResource;
+import io.debezium.configserver.util.Infrastructure;
+import io.debezium.configserver.util.PostgresInfrastructureTestProfile;
+import io.debezium.testing.testcontainers.ConnectorConfigurationTestingHelper;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.startsWith;
+
+@QuarkusTest
+@TestProfile(PostgresInfrastructureTestProfile.class)
+public class ValidatePostgresFiltersIT {
+
+    @Test
+    public void testEmptyPostgresFilters() {
+        ObjectNode config = ConnectorConfigurationTestingHelper.getConfig(
+            Infrastructure.getPostgresConnectorConfiguration(1)
+                .with("database.hostname", "localhost")
+                .with("database.port", Infrastructure.getPostgresContainer().getMappedPort(5432))
+        );
+
+        given().when().contentType(ContentType.JSON).accept(ContentType.JSON).body(config.toString())
+            .post(ConnectorResource.API_PREFIX + ConnectorResource.FILTERS_VALIDATION_ENDPOINT, "postgres")
+            .then().log().all()
+            .statusCode(200)
+            .assertThat().body("status", equalTo("VALID"))
+                .body("propertyValidationResults.size()", is(0))
+                .body("matchedCollections.size()", is(6))
+                .body("matchedCollections",
+                    hasItems(
```

Comment:
If you feel adventureous, how about using text blocks in tests, as [described here](https://www.morling.dev/blog/using-java-13-text-blocks-for-tests/)?

Reply:
Nice blog post! Thank you for that. Unfortunately our Java version is set to 11 not 13.